packages/components/src/Heading/Heading.spec.tsx

```diff
@@ -9,14 +9,15 @@ describe("<Heading />", () => {
   generateSnapshots(HeadingStoryFile);
 
   it("should pass through ref", () => {
-    const headingRef = React.createRef();
+    const headingRef = React.createRef<HTMLHeadingElement>();
     render(
       <Heading ref={headingRef} size="h1" tabIndex={-1}>
         Some Heading
       </Heading>,
     );
 
-    headingRef.current.focus();
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    headingRef.current!.focus();
```

Comment:
It's kind of subtle, but notice the non-null assertion - `!`.

I think in tests it's perfectly reasonable to use them, since the test would fail if `headingRef.current` happens to be missing.